control_custom_device_labview_support_common

```diff
@@ -0,0 +1,15 @@
+Package: ni-veristand-{veristand_version}-custom-device-labview-support-common
+Version: {nipkg_version}
+Architecture: windows_all
+Maintainer: National Instruments <support@ni.com>
+XB-Plugin: file
+Description: Provides common LabVIEW support for custom devices for NI VeriStand {veristand_version}.
+XB-Eula: eula-ni-standard
+Priority: standard
+Homepage: http://www.ni.com
+XB-LanguageSupport: en
+XB-UserVisible: yes
```

Comment:
@buckd @Karl-G1 should this package be user visible? I am leaning towards no.

Reply:
The difficulty with this is how we remove it if all of the packages that need it are uninstalled. Without it being user visible, I'm not sure how we remove it unless we run a `customExecutes` that checks for anything existing in that directory every time we uninstall a package with files in this location.

Let's think about it, but for now, we can leave it as user visible.

Reply:
I believe NIPM automatically removes non-user visible packages when they are no longer referenced.

Reply:
If this is true, then the package should not be user-visible.